checkout.js

```diff
@@ -1,5 +1,9 @@
-function selectSaleItems(items) {
-  return items
+function selectSaleItems(item) {
+  return item.discount > 0
 }
 
-module.exports = selectSaleItems
+
+function filterbyDiscount(items) {
+  return items.filter(selectSaleItems)
+}
+module.exports = filterbyDiscount
```

Comment:
What if you left `selectSaleItems` exported and called `items.filter()` within that function? Then you could have created a second function if you wanted to for doing the `item.discount > 0` and named that something else. While this works, you are now exporting a function called `filterbyDiscount` and then importing it as `selectSaleItems` in your tests which will definitely cost another dev a lot of heartache when they can't figure out what is going on.